frontend/src/pages/spotnet/home/home.css

```diff
@@ -1,3 +1,17 @@
+@import url('https://fonts.googleapis.com/css2?family=Rethink+Sans:ital,wght@0,400..800;1,400..800&display=swap');
+:root {
+  --text-font: 'Rethink Sans', sans-serif;
+  --gradient: linear-gradient(73deg, #74d6fd 1.13%, #e01dee 103.45%);
+  --secondary-color: #1a1c24;
+  --blue-color: #74d6fd;
+  --primary-color: #0b0c10;
+  --black: #000;
+  --primary: #fff;
+  --secondary: #e7ecf0;
+  --button-gradient: linear-gradient(55deg, #74d6fd 0%, #e01dee 100%);
+  --button-gradient-hover: linear-gradient(55deg, #74d6fd 0%, #74d6fd 100%);
+  --button-gradient-active: linear-gradient(55deg, #58c4ef 0%, #58c4ef 100%);
+}
```

Comment:
this seems to partially repeat functionality that already exists in `globals.css`, please combine missing parts into `globals.css` and remove this from `home.css` and verify everything still works correctly